modules/ding_news/ding_news.features.field_instance.inc

```diff
@@ -523,7 +523,7 @@ function ding_news_field_default_field_instances() {
     'display' => array(
       'alternative_layout_full' => array(
         'label' => 'hidden',
-        'module' => 'ting_reference',
+        'module' => 'ting_reference_advanced',
```

Comment:
This is weird.. Did you branch out from ddbcms and not development?

Reply:
Yes, it was branched from development branch.

Reply:
These changes does not come from development branch. They come from modules activated afterwards.